Dockerfile

```diff
@@ -1,40 +1,52 @@
-FROM debian:buster
-
-USER root
-RUN apt-get update -y && \
-	apt-get -y install build-essential libirrlicht-dev cmake libbz2-dev libpng-dev libjpeg-dev \
-		libsqlite3-dev libcurl4-gnutls-dev zlib1g-dev libgmp-dev libjsoncpp-dev git
+FROM alpine:3.11
+
+ENV MINETEST_GAME_VERSION master
+
+COPY .git /usr/src/minetest/.git
+COPY CMakeLists.txt /usr/src/minetest/CMakeLists.txt
+COPY README.md /usr/src/minetest/README.md
+COPY minetest.conf.example /usr/src/minetest/minetest.conf.example
+COPY builtin /usr/src/minetest/builtin
+COPY cmake /usr/src/minetest/cmake
+COPY doc /usr/src/minetest/doc
+COPY fonts /usr/src/minetest/fonts
+COPY lib /usr/src/minetest/lib
+COPY misc /usr/src/minetest/misc
+COPY po /usr/src/minetest/po
+COPY src /usr/src/minetest/src
+COPY textures /usr/src/minetest/textures
```

Comment:
all these `COPY` instructions look prone to breakage, is `COPY . /usr/src/minetest` not possible?

Reply:
it's possible but it copies folders we don't want when local build is in RUN_IN_PLACE and the android build files, the build folders from cmake etc, it's junk for the build.
The main problem is the games folder which generally contains a minetest_game (like in mine) and here break the build on the git clone

Reply:
`/usr/src/minetest` isn't left in the final image, is it?
And for minetest_game it should be possible not to clone it, if it already exists.